scripts/zones/Sea_Serpent_Grotto/mobs/Glyryvilu.lua

```diff
@@ -0,0 +1,14 @@
+-----------------------------------
+-- Area: Sea Serpent Grotto
+--  Mob: Glyryvilu
+-- Note: Popped by qm1
+-- !pos 135x -9y 220z
```

Comment:
`!pos 135 -9 220`

this way someone can copy/paste the command into their game to warp there